STM32F405RGTx/Core/Src/main.cpp

```diff
@@ -136,6 +136,18 @@ int main(void)
   MX_TIM3_Init();
   MX_TIM7_Init();
   /* USER CODE BEGIN 2 */
+
+  // Start PWM timers
+  HAL_TIM_PWM_Start(&htim1, TIM_CHANNEL_1);
```

Comment:
Can you remove these? Starting and stopping the PWM will be handled by the state machine.